front/lib/tracking/customerio/server.ts

```diff
@@ -264,7 +261,7 @@ export class CustomerioServerSideTracking {
         email: user.email,
         first_name: user.firstName,
         last_name: user.lastName,
-        created_at: Math.floor(user.createdAt / 1000),
+        created_at: Math.floor(user.createdAt.getTime() / 1000),
```

Comment:
IMHO we should have a toCustomerIo function on the `UserResource`.

Reply:
Done